cmd/gitstrap/main.go

```diff
@@ -55,6 +61,33 @@ func main() {
 	}
 }
 
+func getToken(flagToken, savedToken string) string {
+	if flagToken != "" {
+		return flagToken
+	}
+	if savedToken != "" {
```

Comment:
@vassilevsky and here we'll be able to return error with some message to show it to user - it may contain some important details, e.g. token file doesn't exist or similar